tests/cli_test.py

```diff
@@ -320,13 +320,13 @@ def test_invalid_calendar(runner):
 def test_attach_calendar(runner):
     runner = runner(days=2)
     result = runner.invoke(main_khal, ['printcalendars'])
-    assert set(result.output.split('\n')[:3]) == set(['one', 'two', 'three'])
+    assert set(result.output.split('\n')[:3]) == {['one', 'two', 'three']}
```

Comment:
Looks like this one's broken.
```suggestion
    assert set(result.output.split('\n')[:3]) == {'one', 'two', 'three'}
```